content/en/docs/tasks/administer-cluster/namespaces.md

```diff
@@ -189,7 +189,7 @@ This delete is asynchronous, so for a time you will see the namespace in the `Te
     To demonstrate this, let's spin up a simple Deployment and Pods in the `development` namespace.
 
     ```shell
-    kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname  -n=development
+    kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname  -n=development --replicas=2
     kubectl scale deployment snowflake --replicas=2 -n=development
```

Comment:
Why this change?
I don't understand it. I'm imagining that the reader will wonder why they are scaling _snowflake` from 2 to 2.

Reply:
good catch
/lgtm cancel

Reply:
Sorry, I forgot to submit new changes. In v1.19, `kubectl create deployment` can already specify replicas and env, hoping to improve the use experience.